src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -404,6 +454,13 @@ Env::meta()
     return item.second;
 }
 
+std::shared_ptr<STTx const>
+Env::tx()
+{
+    auto const item = current()->txRead(txid_);
+    return item.first;
+}
```

Comment:
This will be slightly more efficient if you drop the `const` and `return std::move(item.first)`.  It will avoid two atomic reference count operations.  Not a big deal, but every little bit helps.